tests/check_framework/test_urls.py

```diff
@@ -116,3 +116,33 @@ def test_check_non_unique_namespaces(self):
     def test_check_unique_namespaces(self):
         result = check_url_namespaces_unique(None)
         self.assertEqual(result, [])
+
+
+class CheckURLSettingsTests(SimpleTestCase):
+
+    @override_settings(STATIC_URL='a/', MEDIA_URL='b/')
+    def test_slash_no_errors(self):
+        self.assertEqual(check_url_settings(None), [])
+
+    @override_settings(STATIC_URL='', MEDIA_URL='')
+    def test_empty_string_no_errors(self):
+        self.assertEqual(check_url_settings(None), [])
+
+    @override_settings(STATIC_URL='noslash')
+    def test_static_url_no_slash(self):
+        result = check_url_settings(None)
+        self.assertEqual(len(result), 1)
```

Comment:
The URL tests got started off on a bad foot, I think. I prefer the pattern used in `test_security`. For one thing, if this first assertion fails, you have to use print statement debugging to figure out what the result actually was as opposed to the assertion error giving some useful info.